src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -292,41 +278,24 @@ function sampleDiscussBook(imageFileName) {
   stream.write(request);
 }
 
-require(`yargs`)
+// [END turing_prog_request_streaming_bidi_core]
+// tslint:disable-next-line:no-any
+// [END turing_prog_request_streaming_bidi]
+argv = require(`yargs`)
   .default('image_file_name', 'image_file.jpg')
   .argv;
 
 sampleDiscussBook(argv.image_file_name);
-*/
-// [END turing_prog_request_streaming_bidi_core]
-
-// FIXME: Insert here clean-up code.
-// tslint:disable-next-line:no-any
-// [END turing_prog_request_streaming_bidi]
 ============== file: src/samples/v1/find_related_books_request_async_paged_all_odyssey.js ==============
-//// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "FindRelatedBooksRequestAsyncPagedAllOdyssey" ]
-//// STUB standalone sample "FindRelatedBooksRequestAsyncPagedAllOdyssey" /////
-
-// FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
-
+// DO NOT EDIT! This is a generated sample ("RequestAsyncPagedAll",  "odyssey")
+'use strict';
 // [START sample]
-
-// FIXME: Insert here boilerplate code not directly related to the method call itself.
-
-//      calling form: "RequestAsyncPagedAll"
-//        region tag: "sample"
-//         className: "FindRelatedBooksRequestAsyncPagedAllOdyssey"
-//          valueSet: "odyssey" ("The Odyssey")
-//       description: "Testing calling forms"
-//        [names[0]=Odyssey, shelves[0]=Classics]
-//      apiMethod "findRelatedBooks" of type "PagedOptionalArrayMethod"
-
 // [START sample_core]
-
-// FIXME: Insert here code to prepare the request fields, make the call, process the response.
-
-/*
 function sampleFindRelatedBooks() {
+  const library = require('@google-cloud/library');
+  const client = new library.v1.LibraryServiceClient({
```

Comment:
One concern I have about this is that in JavaScript creating a new client is an expensive operation that involves reading and parsing proto files. By creating a client here we have a risk that users will copy this function as it goes and will create a lot of clients in their code instead of creating just one. I have some feeling that it might be better if a function accepted a client as a parameter. @beccasaurus what do you think?

Reply:
> creating a new client is an expensive operation

Our code samples are isolated snippets which should be copy/paste runnable.

In cases where creating a new client object would hurt the user, we can call those out with custom hand-written code samples, e.g. to demonstrate re-using service objects to preserve streams/etc.

This follows the existing conventions of all code samples – 👍 we should instantiate client object